packages/recomponents/src/components/r-button/r-button.spec.js

```diff
@@ -119,4 +119,14 @@ describe('r-button.vue', () => {
         expect(wrapper.classes()).toContain('r-button-fluid');
         expect(wrapper.attributes().disabled).not.toBe(undefined);
     });
+
+    it('should render component with capitalizing the first letter', () => {
```

Comment:
👍 I would add the same test to` r-icon-button.spec.js`

Reply:
added :-)

Reply:
With CSS only solution no need to update tests I guess

Reply:
yes, no need unit tests if CSS solution works, that's why I prefer another PR